lib/benchpark/system.py

```diff
@@ -72,6 +73,13 @@ class System(ExperimentSystemBase):
         Tuple["benchpark.variant.Variant", "benchpark.spec.ConcreteSystemSpec"],
     ]
 
+    variant(
+        "scheduler",
+        default="yes",
+        values=("yes", "none"),
+        description="Use the scheduler on the system, or generate scripts without the scheduler commands",
+    )
```

Comment:
```suggestion
    variant(
        "scheduler",
        default="mpi",
        values=("mpi",),
        description="Use the scheduler on the system, or generate scripts without the scheduler commands",
    )
```

Reply:
I reverted the changes from @dyokelson.  @becker33 the dry run now shows the error: the value of the variant is 'mpi' when the concretizer expects 'None'.